site/content/blog/kctrl-app-commands-blog.md

```diff
@@ -0,0 +1,509 @@
+---
+title: "Getting to know App resources better with kctrl"
+slug: kcrl-app-commands-blog
+date: 2022-04-28
+author: Soumik Majumder
+excerpt: "Get better visibility into your app resources with kctrl's new app commands and enhancements to the package commands"
+image: /img/kapp-controller.svg
+tags: ['carvel', 'kapp-controller', 'gitops', 'apps', 'kctrl', 'cli']
+---
+
+Since the previous release of `kctrl`, we have been iterating over how we can help users take a closer look at what App CRs are doing on the cluster. This includes ones authored by users and those that are created as a result of package installations.
+
+As promised, the latest release of `kctrl` introduces a set of commands which allow users to observe and interact with App CRs more conveniently. `kctrl` v0.36.0 also introduces an improved status tailing experience which surfaces relevant information while waiting for resources to reconcile.
+
+# The `app` commands
+These commands aim to help users manage App CRs on the cluster better. Lets dig in and see what they look like!
+
+The user can list apps created on the cluster by running,
+```bash
+$ kctrl app list
+Target cluster 'https://34.135.209.41' (nodes: gke-cluster-sm-default-pool-28d3ddcd-gjqs, 2+)
+
+Available apps in namespace 'default'
+
+Name        Status               Since Deploy  Owner                     Age  
+hello-app   Reconcile succeeded  1m            PackageInstall/hello-app  1m  
+simple-app  Reconcile succeeded  22s           -                         1m  
+
+Succeeded
+```
+`kctrl` makes a distinction between App CRs deployed by themselves and the ones created due to a package installation. We can take a closer look at an app by using the `get` command.
```

Comment:
nit:  App CRs deployed by `users` and the ones ...

instead of  

App CRs deployed by `themselves` and the ones...

Reply:
I think this(`We can take a closer look at an app by using the `get` command.`) can be in next line as we are changing the context from list to get.

Reply:
Makes sense to me ! making the changes